pulumi/bin/copy_artifacts_from_rc.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+# This will grab a subset of keys from `origin/rc`'s Pulumi.testing.yaml.
+set -euo pipefail
+
+readonly THIS_DIR=$(dirname "${BASH_SOURCE[0]}")
+
+ensureRightDir() {
+    cd $THIS_DIR/../grapl
+}
+ensureRightDir
+
+readonly GRAPL_ROOT="$(git rev-parse --show-toplevel)"
+readonly RC_CONFIG_FILE="/tmp/rc_pulumi_testing.yaml"
+readonly CURRENT_STACK="$(pulumi stack --show-name)"
```

Comment:
I think there are cases where there is no currently selected stack; it would probably be better to pass in a name as an argument.